client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -42,14 +42,15 @@ const UserPage = createClass({
 	renderBrews : function(brews){
 		if(!brews || !brews.length) return <div className='noBrews'>No Brews.</div>;
 
-		const sortedBrews = this.sortBrews(brews, this.state.sortType);
+		const sortedBrews = this.sortBrews(brews);
 
 		return _.map(sortedBrews, (brew, idx)=>{
 			return <BrewItem brew={brew} key={idx}/>;
 		});
 	},
 
 	sortBrewOrder : function(brew){
+		if(!brew.title){brew.title = 'No Title';};
 		const mapping = {
 			'alpha'   : _.deburr(brew.title.toLowerCase()),
```

Comment:
```suggestion
		const mapping = {
			'alpha'   : _.deburr(brew.title?.toLowerCase() || "No Title"),
```

How about this?

Reply:
Unfortunately that change results in the following Brew entry in the User Page:

![image](https://user-images.githubusercontent.com/19826659/128104650-a4988c62-f103-41ba-bef9-d27f12f0e982.png)

Intention is to remain with the original code at this time.